applications/mne_scan/plugins/hpi/hpi.cpp

```diff
@@ -1,589 +1,615 @@
-//=============================================================================================================
```

Comment:
What happened here? Or is this just some git related change which is not actually change? Did you replace the files? Seems a lot of files in this PR are affected. Any way this can be reverted and only include the MNE_TRACE cahnges? It will overwrite the history and make using git blame difficult in the future.

Reply:
@juangpc Anyway you can get rid of these complete file replace changes? As I see it, the only changes to these files the MNE tracing macro calls.